include/dd/Edge.hpp

```diff
@@ -33,8 +33,8 @@ namespace dd {
         [[nodiscard]] constexpr bool isTerminal() const { return Node::isTerminal(p); }
 
         // edges pointing to zero and one terminals
-        static inline Edge one{Node::terminal, Complex::one};
-        static inline Edge zero{Node::terminal, Complex::zero};
+        static const inline Edge one{Node::terminal, Complex::one};   // NOLINT(readability-identifier-naming) automatic renaming does not work reliably, so skip linting
```

Comment:
does the const really work here? I believe I had troubles with that in the past in our top-level repos.

Reply:
DDSIM builds and tests are successful, so we'll keep it const :)